model.py

```diff
@@ -140,7 +140,9 @@ def __init__(self, *args, func_norm=poly, IP_hs=50, xcen=0):
         self.func_norm = func_norm
         #print("sampling [km/s]:", self.dx*c)
 
-    def __call__(self, pixel, rv, coeff_norm, coeff_wave, coeff_ip, coeff_atm=[], coeff_bkg=[0], coeff_ipB=[]):
+    def __call__(self, pixel, rv=0, norm=[1], wave=[], ip=[], atm=[], bkg=[0], ipB=[]):
+        # renaming (coeff is ok prefix below, but too verbose for par)
```

Comment:
The renaming of the arguments inside in the model call is not so nice. But it was easiest for now.

Passing named parameter will make things easier. More to come.